### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by HaiShaw**
## Motivation

As it is

## Checklist

- [+] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [+] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [+] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

**Comment by zhyncs** [comment]
Please fix lint issue

File: test/srt/test_eval_accuracy_large.py [hunk 1]
```diff
@@ -68,6 +68,20 @@ def test_mgsm_en(self):
         metrics = run_eval(args)
         self.assertGreater(metrics["score"], 0.835)
 
+    def test_math(self):
```
Comment (merrymercy) [inline test/srt/test_eval_accuracy_large.py]: revert the change of these two files.

**Comment by HaiShaw** [comment]
@merrymercy @zhyncs what is going on, lint failed on these 2 files, that I did not touch originally.

**Comment by merrymercy** [comment]
rebase to the main, the buggy commit is reverted  https://github.com/sgl-project/sglang/commit/32318178611b2cbbeddb272e8388ea25f4b64cf5

**Comment by HaiShaw** [comment]
@merrymercy done.
